src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="search_container"
+      action="/search"
+      method="get"
+    >
+      <img
+        src="./images/Search.svg"
+        alt="magnifier"
+        class="search_icon--1"
```

Comment:
```suggestion
        class="search_icon"
```
![Screenshot from 2023-09-07 13-36-17](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/c3a82222-67fb-461b-8364-227c4984635f)